CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.1.0
+
+* Migrate to Android V2
+* upgrade Flutter SDK version for versions < 4.0.0
```

Comment:
```suggestion
* Upgrade Flutter SDK version for versions < 4.0.0
```

Reply:
To be compatible with Dart 3 features, we upgrade the Dart SDK version with `sdk: '>=2.12.0 <4.0.0'`